### PR TITLE
DOC: Sequential pipelines doc

### PR DESCRIPTION
**Pull request by Shubham9t9**
# Description

Added steps to create a sequential pipelines.
Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# Checklist:

* [X] The title of the PR states what changed and the related issues number (used for the release note).
* [X] Does this PR require documentation updates?
* [X] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have tested it for all user roles

### CODE REVIEW

File: docs/user-guide/creating-application/workflow/cd-pipeline.md [hunk 1]
```diff
@@ -216,5 +219,10 @@ It terminates the old version and releases the new one.
 
 Devtron now supports attaching multiple deployment pipelines to a single build pipeline, in its workflow editor. This feature lets you deploy an image first to stage, run tests and then deploy the same image to production.
 
+Please follow the steps mentioned below to create sequential pipelines :
+
+After creating `CI pipeline` or `build pipeline`, create a CD pipeline by clicking on the `+` sign on CI pipeline and configure the CD pipeline as per your requirements.
```
[the remark this answers is not in the document]
Comment (Shubham9t9) [reply docs/user-guide/creating-application/workflow/cd-pipeline.md]: Corrected

File: docs/user-guide/creating-application/workflow/cd-pipeline.md [hunk 2]
```diff
@@ -216,5 +219,10 @@ It terminates the old version and releases the new one.
 
 Devtron now supports attaching multiple deployment pipelines to a single build pipeline, in its workflow editor. This feature lets you deploy an image first to stage, run tests and then deploy the same image to production.
 
+Please follow the steps mentioned below to create sequential pipelines :
 
-![](../images/sequential-workflow.jpg)
\ No newline at end of file
+After creating CI/build pipeline, create a CD pipeline by clicking on the `+` sign on CI pipeline and configure the CD pipeline as per your requirements.
+To create CD pipeline for some other environment, click on `+` sign on CD pipeline.
```
Comment (prakarsh-dt) [inline docs/user-guide/creating-application/workflow/cd-pipeline.md:225]: To add another CD Pipeline sequentially after previous one, again click on `+` sign on the last CD pipeline block.

File: docs/user-guide/creating-application/workflow/cd-pipeline.md [hunk 3]
```diff
@@ -216,5 +219,10 @@ It terminates the old version and releases the new one.
...
+You can add CD pipelines for multiple environments in the sequential pipelines. Also you can add CD pipeline before or after any existing CD pipeline just by clicking `+` sign on CD pipeline according to the required sequence.
```
Comment (prakarsh-dt) [inline docs/user-guide/creating-application/workflow/cd-pipeline.md:226]: Similarly, you can add multiple CD pipelines by clicking `+` sign of the last CD pipeline, each deploying in different environments.

File: docs/user-guide/creating-application/workflow/cd-pipeline.md [hunk 4]
```diff
@@ -216,5 +219,10 @@ It terminates the old version and releases the new one.
...
+
```
Comment (prakarsh-dt) [inline docs/user-guide/creating-application/workflow/cd-pipeline.md:227]: Add
> Note: Deleting a CD pipeline also deletes all the K8s resources associated with it and will bring a disruption in the deployed micro-service. Before deleting a CD pipeline, please ensure that the associated resources are not being used in any production workload.

File: docs/user-guide/creating-application/workflow/cd-pipeline.md (hunk 2 above)
Comment (Shubham9t9) [reply docs/user-guide/creating-application/workflow/cd-pipeline.md]: Edited

File: docs/user-guide/creating-application/workflow/cd-pipeline.md (hunk 3 above)
Comment (Shubham9t9) [reply docs/user-guide/creating-application/workflow/cd-pipeline.md]: Edited

File: docs/user-guide/creating-application/workflow/cd-pipeline.md (hunk 4 above)
Comment (Shubham9t9) [reply docs/user-guide/creating-application/workflow/cd-pipeline.md:227]: Added

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**